packages/gatsby/src/redux/actions/internal.ts

```diff
@@ -47,22 +47,6 @@ export const createPageDependency = (
   }
 }
 
-/**
- * Delete dependencies between an array of pages and data. Probably for
- * internal use only. Used when deleting pages.
- * @private
- */
-export const deleteComponentsDependencies = (
-  paths: Array<string>
-): IDeleteComponentDependenciesAction => {
-  return {
-    type: `DELETE_COMPONENTS_DEPENDENCIES`,
-    payload: {
-      paths,
-    },
-  }
-}
-
```

Comment:
Not needed anymore because we use the new action `QUERY_START` to consistently reset data dependencies before running a query.